### PR TITLE
Implemented pnpm and fixed tests

### PR DESCRIPTION
**Pull request by iTweeno**

### CODE REVIEW

**Review by AshleyDawson — CHANGES_REQUESTED** [review]
Please update README.md docs to reflect use of pnpm

File: package.json [hunk 1]
```diff
@@ -89,4 +90,4 @@
     "test": "jest",
     "lint": "yarn eslint --ext ts,tsx ./src"
```
Comment (zpawn) [inline package.json:91]: Change scripts to use `pnpm`

**Comment by zpawn** [comment]
I think need to remove the `yarn.lock`

**Comment by iTweeno** [comment]
@AshleyDawson

**Comment by AshleyDawson** [comment]
> @AshleyDawson

@iTweeno if you hit the "re-review" button I'll get an alert :) 

image

**Review by AshleyDawson — APPROVED** [review]

**Comment by iTweeno** [comment]
Am dumb lol

**Pull request merged**